relate/settings.py

```diff
@@ -14,9 +14,11 @@
 from django.conf.global_settings import STATICFILES_FINDERS, gettext_noop
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import sys
 import os
 from os.path import join
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+sys.path.append(BASE_DIR)
```

Comment:
Can we do without this?

Reply:
Can you spell it out? :-)

Reply:
Opooos, I didn't know that's possible....